src/configs/shell.rs

```diff
@@ -13,6 +13,7 @@ pub struct ShellConfig<'a> {
     pub ion_indicator: &'a str,
     pub elvish_indicator: &'a str,
     pub tcsh_indicator: &'a str,
+    pub default: &'a str,
```

Comment:
Personally I would prefer to use a more specific name for this option maybe `unknown_indicator`?

Reply:
Sure, that certainly makes more sense! I'll change it.